olm-catalog/serverless-operator/generate_csv.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+template="${1:?Provide template file as arg[1]}"
+target="${2:?Provide a target CSV file as arg[2]}"
+
+source "$(dirname "${BASH_SOURCE[0]}")/../../hack/lib/vars.bash"
+
+registry="registry.svc.ci.openshift.org/openshift"
+serving="$registry/knative-$KNATIVE_SERVING_VERSION:knative-serving"
+eventing="$registry/knative-$KNATIVE_EVENTING_VERSION:knative-eventing"
+
+declare -A images
+images=(
+  ["queue-proxy"]="$serving-queue"
+  ["activator"]="$serving-activator"
```

Comment:
This might be a bit confusing, as people might take this as a _variable_ name "serving-queue". It's not allowed in bash, but I'm not so sure about other shells. I think its better to be more explicit and use "${serving}-queue" instead. That makes the intention crystal clear.

Reply:
Done.